WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -157,7 +157,7 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 
 			if (!registeredAliceClientAndCircuits.Any())
 			{
-				Logger.LogInfo($"Round ({roundId}): There is no available alices to participate with.");
+				Logger.LogInfo($"Round ({roundId}): There are no available alices to participate with.");
```

Comment:
```suggestion
				Logger.LogInfo($"Round ({roundId}): There are no available Alices to participate with.");
```

Reply:
+1, but we have a `LogCoinJoinSummary` method, maybe we could log the TxID there.